ui/apps/platform/src/Containers/Policies/Wizard/Step1/MitreTacticSelect.tsx

```diff
@@ -43,6 +43,7 @@ function MitreTacticSelect({
             onToggle={(_event, val) => setIsOpen(val)}
             placeholderText={label}
             selections={tacticId}
+            maxHeight="300px"
```

Comment:
If these issues are related to dropdowns within a Wizard, should we consider a CSS solution that targets select components within wizard components and adds a max height to those? 

I know we will eventually replace the deprecated Select, so if it's unnecessary extra work, that is ok.

Reply:
Yeah this is a really good point, and the reason I went with a quick value in all locations was because these are going to be replaced short term.

This actually might extend beyond the wizard. I imagine any Select with an unbounded number of items will eventually scroll off the page, or be otherwise difficult to use. This might be fixed in the newer version? 🤔 

Either way, when we migrate these it makes sense to see if a global or nearly global rule on maxHeight is something we should implement.